### PR TITLE
openloops: add check for added Fortran compiler

### PR DESCRIPTION
**Pull request by bernhardkaindl**
Add a check for `self.compiler.fc` like in `openmpi` and `openblas` before using `SPACK_FC`

### CODE REVIEW

**Comment by bernhardkaindl** [comment]
@spackbot run-pipeline

**Comment by bernhardkaindl** [comment]
@spackbot help

**Comment by bernhardkaindl** [comment]
@spackbot run pipeline

**Review by iarspider — APPROVED** [review]

**Comment by iarspider** [comment]
Thanks!

**Comment by bernhardkaindl** [comment]
> Thanks!

@iarspider: You can enable me to merge it when you click "Add your review" on the right of "Review required", and then select "Approve" in the window after that and then submit the Approve.

You can also open the same Review dialog in the "Files" tab in the right upper corner.

**Review by iarspider — DISMISSED** [review]

**Comment by iarspider** [comment]
> > Thanks!
> 
> @iarspider: You can enable me to merge it when you click "Add your review" on the right of "Review required", and then select "Approve" in the window after that and then submit the Approve.
> 
> You can also open the same Review dialog in the "Files" tab in the right upper corner.

I can't - I don't have merge access :)

File: var/spack/repos/builtin/packages/openloops/package.py
Comment (alalazo) [inline var/spack/repos/builtin/packages/openloops/package.py]: Do you mean:
```suggestion
                raise InstallError(f"{self.spec.compiler} has no Fortran compiler in spack!")
```
?

**Review by alalazo — APPROVED** [review]

**Comment by bernhardkaindl** [comment]
@spackbot run-pipeline

**Comment by bernhardkaindl** [comment]
@spackbot help

**Comment by bernhardkaindl** [comment]
@spackbot run pipeline

**Pull request merged**
